test/tools/WebListener/WebListener.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.0-*" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.1.0-rc1-30676" />
```

Comment:
Maybe we should use "Microsoft.AspNetCore.App"?
https://andrewlock.net/exploring-the-microsoft-aspnetcore-app-shared-framework-in-asp-net-core-2-1-preview-1/

Reply:
Thanks for the link, will update